packages/notifi-react-card/lib/components/subscription/EventTypeFusionHealthCheckRow.tsx

```diff
@@ -99,6 +100,9 @@ export const EventTypeFusionHealthCheckRow: React.FC<
   const UNABLE_TO_SUBSCRIBE = 'Unable to subscribe, please try again';
   const UNABLE_TO_UNSUBSCRIBE = 'Unable to unsubscribe, please try again';
   const INVALID_NUMBER = 'Please enter a valid number';
+  const INVALID_RANGE = (validInputRange: ValidInputRange) => {
+    return `Please enter a valid number (${validInputRange.min} - ${validInputRange.max})`;
```

Comment:
is this copy okay? maybe something like "Please enter a number between {min} and {max}"